app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/importlogins/ImportLoginsScreen.kt

```diff
@@ -0,0 +1,240 @@
+                confirmButtonText = confirmButtonText,
+                dismissButtonText = dismissButtonText,
+                onConfirmClick = handler.onConfirmImportLater,
+                onDismissClick = handler.onDismissDialog,
+            )
+        }
+
+        null -> Unit
+    }
+}
+
+@Composable
+private fun ImportLoginsContent(
+    onGetStartedClick: () -> Unit,
+    onImportLaterClick: () -> Unit,
+) {
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .verticalScroll(rememberScrollState())
+            .padding(top = 24.dp)
+            .navigationBarsPadding(),
+        horizontalAlignment = Alignment.CenterHorizontally,
+    ) {
+        Image(
+            painter = rememberVectorPainter(R.drawable.img_import_logins),
+            contentDescription = null,
+            modifier = Modifier
+                .standardHorizontalMargin()
+                .size(124.dp),
+        )
+        Spacer(Modifier.height(24.dp))
+        Text(
+            text = stringResource(R.string.give_your_vault_a_head_start),
+            style = BitwardenTheme.typography.titleMedium,
+            color = BitwardenTheme.colorScheme.text.primary,
+            textAlign = TextAlign.Center,
+            modifier = Modifier.standardHorizontalMargin(),
+        )
+        Spacer(Modifier.height(12.dp))
+        Text(
+            text = stringResource(
+                R.string.from_your_computer_follow_these_instructions_to_export_saved_passwords,
+            ),
+            style = BitwardenTheme.typography.bodyMedium,
+            color = BitwardenTheme.colorScheme.text.primary,
+            textAlign = TextAlign.Center,
+            modifier = Modifier.standardHorizontalMargin(),
+        )
+        Spacer(Modifier.height(24.dp))
+        BitwardenFilledButton(
+            label = stringResource(R.string.get_started),
+            onClick = onGetStartedClick,
+            modifier = Modifier
+                .standardHorizontalMargin()
+                .fillMaxWidth(),
+        )
+        Spacer(Modifier.height(12.dp))
+        BitwardenOutlinedButton(
+            label = stringResource(R.string.import_logins_later),
+            onClick = onImportLaterClick,
+            modifier = Modifier
+                .standardHorizontalMargin()
+                .fillMaxWidth(),
+        )
+    }
+}
+
+@Preview
+@Preview(uiMode = Configuration.UI_MODE_NIGHT_YES)
+@Composable
+private fun ImportLoginsInitialContent_preview() {
+    BitwardenTheme {
+        Surface {
+            ImportLoginsContent(
+                onGetStartedClick = {},
+                onImportLaterClick = {},
+            )
+        }
+    }
+}
+
+@Preview
+@Preview(uiMode = Configuration.UI_MODE_NIGHT_YES)
+@Composable
+private fun ImportLoginsScreenDialog_preview(
+    @PreviewParameter(ImportLoginsDialogContentPreviewProvider::class) state: ImportLoginsState,
+) {
+    BitwardenTheme {
+        Surface {
```

Comment:
What's with the surface?

Reply:
Gives us the background, this is with it removed. I can switch it for Column or something with explicit bg color to save the extra import. LMK what you prefer.
![image](https://github.com/user-attachments/assets/366979d0-bd5f-47a4-9eb9-3a85ea867c93)

Reply:
which after typing that out I realize its just using the Material.surface color under the hood so I guess we don't want that even in previews.

Reply:
How much would we hate a public function `BitwardenPreview(content: () -> Unit)` to save this for previews.

Reply:
Hmmm, we could also make a custom annotation that does a decent but of it for us. 🤔 

Feels like a different PR though, the Surface is fine here.